sdks/python/container/boot.go

```diff
@@ -127,6 +129,10 @@ func main() {
 	// Guard from concurrent artifact retrieval and installation,
 	// when called by child processes in a worker pool.
 
+	if err := setupAcceptableWheelSpecs(); err != nil {
+		log.Fatalf("Failed to setup acceptable wheel specs: %v", err)
```

Comment:
I think we should not fail but leave acceptableWhlSpecs as empty in this case.

Reply:
It's true. We should only output messages even when error returned.